<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MotorControllerConfigWindow</class>
 <widget class="QDialog" name="MotorControllerConfigWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>902</width>
    <height>328</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Controller Configuration</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="7">
      <widget class="QLineEdit" name="editMotorTemp1Offset"/>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>Inv Temp 2 Offset</string>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Current 1 Offset</string>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QLabel" name="label_23">
       <property name="text">
        <string>Motor Type</string>
       </property>
      </widget>
     </item>
     <item row="0" column="7">
      <widget class="QLineEdit" name="editCurrent1Offset"/>
     </item>
     <item row="2" column="4">
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>Motor Temp 1 Scale</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>Inv Temp 1 Scale</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="label_18">
       <property name="text">
        <string>Motor Temp 2 Offset</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Encoder Count</string>
       </property>
      </widget>
     </item>
     <item row="4" column="4">
      <widget class="QLabel" name="label_25">
       <property name="text">
        <string>Pole Pairs</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="editEncoderCount"/>
     </item>
     <item row="0" column="5">
      <widget class="QLineEdit" name="editCurrent1Scale"/>
     </item>
     <item row="4" column="3">
      <widget class="QLineEdit" name="editEncoderDir"/>
     </item>
     <item row="4" column="7">
      <widget class="QLineEdit" name="editRotorTime"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="editCurrent2Scale"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Max Torque</string>
       </property>
      </widget>
     </item>
     <item row="2" column="6">
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string>Motor Temp 1 Offset</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLabel" name="label_19">
       <property name="text">
        <string>Encoder Direction</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Voltage Scale</string>
       </property>
      </widget>
     </item>
     <item row="3" column="6">
      <widget class="QLabel" name="label_24">
       <property name="text">
        <string>Control Type</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLineEdit" name="editVoltageOffset"/>
     </item>
     <item row="1" column="3">
      <widget class="QLineEdit" name="editCurrent2Offset"/>
     </item>
     <item row="1" column="5">
      <widget class="QLineEdit" name="editInvTemp1Scale"/>
     </item>
     <item row="1" column="7">
      <widget class="QLineEdit" name="editInvTemp1Offset"/>
     </item>
     <item row="2" column="3">
      <widget class="QLineEdit" name="editInvTemp2Offset"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Torque KP</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Motor Temp 2 Scale</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Current 1 Scale</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Inv Temp 2 Scale</string>
       </property>
      </widget>
     </item>
     <item row="6" column="6">
      <widget class="QLabel" name="label_30">
       <property name="text">
        <string>VHz Full RPM</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QLineEdit" name="editMotorTemp2Offset"/>
     </item>
     <item row="3" column="5">
      <widget class="QLineEdit" name="editMotorType"/>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="editVoltageScale"/>
     </item>
     <item row="5" column="2">
      <widget class="QLabel" name="label_20">
       <property name="text">
        <string>Torque KI</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="editInvTemp2Scale"/>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Current 2 Offset</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Voltage Offset</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Current 2 Scale</string>
       </property>
      </widget>
     </item>
     <item row="4" column="6">
      <widget class="QLabel" name="label_26">
       <property name="text">
        <string>Rotor Time Const</string>
       </property>
      </widget>
     </item>
     <item row="5" column="4">
      <widget class="QLabel" name="label_27">
       <property name="text">
        <string>Torque KD</string>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QLabel" name="label_21">
       <property name="text">
        <string>Max Amps Drive</string>
       </property>
      </widget>
     </item>
     <item row="1" column="6">
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>Inv Temp 1 Offset</string>
       </property>
      </widget>
     </item>
     <item row="6" column="4">
      <widget class="QLabel" name="label_29">
       <property name="text">
        <string>Max Amps Regen</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="editMotorTemp2Scale"/>
     </item>
     <item row="7" column="2">
      <widget class="QLabel" name="label_22">
       <property name="text">
        <string>Hall AB</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Theta Offset</string>
       </property>
      </widget>
     </item>
     <item row="2" column="5">
      <widget class="QLineEdit" name="editMotorTemp1Scale"/>
     </item>
     <item row="7" column="6">
      <widget class="QLabel" name="label_32">
       <property name="text">
        <string>Hall CA</string>
       </property>
      </widget>
     </item>
     <item row="4" column="5">
      <widget class="QLineEdit" name="editPolePairs"/>
     </item>
     <item row="5" column="6">
      <widget class="QLabel" name="label_28">
       <property name="text">
        <string>Max RPM</string>
       </property>
      </widget>
     </item>
     <item row="7" column="4">
      <widget class="QLabel" name="label_31">
       <property name="text">
        <string>Hall BC</string>
       </property>
      </widget>
     </item>
     <item row="3" column="7">
      <widget class="QLineEdit" name="editControlType"/>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="editTorqueKP"/>
     </item>
     <item row="5" column="3">
      <widget class="QLineEdit" name="editTorqueKI"/>
     </item>
     <item row="5" column="5">
      <widget class="QLineEdit" name="editTorqueKD"/>
     </item>
     <item row="5" column="7">
      <widget class="QLineEdit" name="editMaxRPM"/>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="editMaxTorque"/>
     </item>
     <item row="6" column="3">
      <widget class="QLineEdit" name="editMaxAmpsDrive"/>
     </item>
     <item row="6" column="5">
      <widget class="QLineEdit" name="editMaxAmpsRegen"/>
     </item>
     <item row="6" column="7">
      <widget class="QLineEdit" name="editVHZFull"/>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="editThetaOffset"/>
     </item>
     <item row="7" column="3">
      <widget class="QLineEdit" name="editHallAB"/>
     </item>
     <item row="7" column="5">
      <widget class="QLineEdit" name="editHallBC"/>
     </item>
     <item row="7" column="7">
      <widget class="QLineEdit" name="editHallCA"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnRefresh">
       <property name="text">
        <string>Refresh</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSave">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
